src/main/java/com/example/bookshop/controller/BookController.java

```diff
@@ -19,46 +22,55 @@
 import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
 
+@Tag(name = "Book management", description = "Endpoints for managing books")
 @RequiredArgsConstructor
 @RestController
 @RequestMapping("/books")
 public class BookController {
     private final BookService bookService;
 
+    @Operation(summary = "find all books where authors with certain name",
+                description = "find all books by athours")
     @GetMapping("/author")
     public List<BookDto> findAllByAuthor(@RequestParam String author) {

```

Comment:
```suggestion
    public List<BookDto> findAllByAuthor(@RequestParam String author, @PageableDefault Pageable pageable) {

```
It is better to use Pageable here, since the number of books can be large